src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -26,6 +26,8 @@ class ContentExplorer extends Component {
          * Each column has to be a Column element
          */
         additionalColumns: PropTypes.arrayOf(PropTypes.element),
+        /** Items that will show up as selected */
+        autoSelectedItems: PropTypes.object,
```

Comment:
what are your thoughts on something like `controlledSelectedItems`? to convey that the parent component can force a selection over the user

Reply:
That is more descriptive yes.

Reply:
Updated